.ado/pipelines/config/configuration.yaml

```diff
@@ -1,6 +1,9 @@
 # Central configuration and versioning settings
 # used for all pipelines
 variables:
+- name: 'riVariant' # Reference implementation variant - each repository should have its own identification
```

Comment:
should we add a comment where this is being used?

Reply:
Reading my mind :) I was thinking about that, but then decided not to add it, in case this value is used somewhere else than in the UI. But since it isn't for now, I will add it.